Sources/Testing/Issues/Issue.swift

```diff
@@ -115,9 +115,29 @@ public struct Issue: Sendable {
   @_spi(ForToolsIntegrationOnly)
   public var sourceContext: SourceContext
 
+  /// A type representing a
+  /// ``withKnownIssue(_:isIntermittent:sourceLocation:_:when:matching:)`` call
+  /// that matched an issue.
+  @_spi(ForToolsIntegrationOnly)
+  public struct KnownIssueContext: Sendable, Codable {
```

Comment:
The word "issue" in the name of this type may be redundant now that it's nested within `Issue`. Does `Issue.KnownContext` sound strange? Will ponder it, but curious what you think

Reply:
`Issue.KnownContext` does sound a little strange to me. I do agree the current name is a bit repetitive. I'd actually be tempted to un-nest this type and just call it `KnownIssue` to match `withKnownIssue`. I.e., `withKnownIssue` will apply a `KnownIssue` to any `Issue`s that are recorded by its closure.

Reply:
Between those options I think I'd prefer to leave it nested with the name you have right now. We can consider it more deeply if/when we decide to elevate this to fully public API.